codespell_lib/data/dictionary.txt

```diff
@@ -22779,7 +22786,7 @@ synomym->synonym
 synonim->synonym
 synonomous->synonymous
 synonymns->synonyms
-synopsys->synopsis
+synopsys->synopsis, disabled since this is also a company name
```

Comment:
Similar to surnames, but probably worse, companies have a terrible habit of misspelling stuff, as someone has already taken the correct spelling, or because they want to look trendy.

Reply:
That's true, although again I'd argue that the number of large tech companies that may be mentioned in source code with names that are also misspellings is not too large.

Reply:
I don't think I'd ever personally heard of this company.

I feel companies are a weaker reasoning as they don't pop up unexpectedly/the chances of "them" contributing is probably significantly lower, and therefore white-listing seems more appropriate.

Reply:
(Dropping names including this company name from this CL)

Reply:
Thanks